tests/unit/amo/components/TestUserProfileEdit.js

```diff
@@ -1119,6 +1131,7 @@ describe(__filename, () => {
 
     expect(root).toHaveState('picture', null);
     expect(root).toHaveState('pictureData', null);
+    sinon.assert.calledWith(_window.scroll, (0, 0));
```

Comment:
@willdurand we need to do same here as above :)

Reply:
oh right, I missed this one, thanks!